File: services/src/main/java/com/liferay/mobile/sdk/v7/ratingsentry/RatingsEntryService.java
package com.liferay.mobile.sdk.v7.ratingsentry;

import com.liferay.mobile.sdk.Call;
import com.liferay.mobile.sdk.annotation.Param;
import com.liferay.mobile.sdk.annotation.Path;
import com.liferay.mobile.sdk.http.Response;
import java.lang.String;
import org.json.JSONObject;

@Path("/ratingsentry")
public interface RatingsEntryService {
  @Path("/delete-entry")
  Call<Response> deleteEntry(@Param(name = "className") String className, @Param(name = "classPK") long classPK);

  @Path("/update-entry")
  Call<JSONObject> updateEntry(@Param(name = "className") String className, @Param(name = "classPK") long classPK, @Param(name = "score") double score);
}

File: windows/Liferay.SDK/Service/V72/RatingsEntry/RatingsEntryService.cs
/**
 * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
 *
 * This library is free software; you can redistribute it and/or modify it under
 * the terms of the GNU Lesser General Public License as published by the Free
 * Software Foundation; either version 2.1 of the License, or (at your option)
 * any later version.
 *
 * This library is distributed in the hope that it will be useful, but WITHOUT
 * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
 * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
 * details.
 */

using System;
using System.Collections.Generic;
using System.IO;
using System.Linq;
using System.Threading.Tasks;

namespace Liferay.SDK.Service.V72.RatingsEntry
{
	public class RatingsEntryService : ServiceBase
	{
		public RatingsEntryService(ISession session)
			: base(session)
		{
		}

		public async Task DeleteEntryAsync(string className, long classPK)
		{
			var _parameters = new JsonObject();

			_parameters.Add("className", className);
			_parameters.Add("classPK", classPK);

			var _command = new JsonObject()
			{
				{ "/ratingsentry/delete-entry", _parameters }
			};

			await this.Session.InvokeAsync(_command);
		}

		public async Task<dynamic> UpdateEntryAsync(string className, long classPK, double score)
		{
			var _parameters = new JsonObject();

			_parameters.Add("className", className);
			_parameters.Add("classPK", classPK);
			_parameters.Add("score", score);

			var _command = new JsonObject()
			{
				{ "/ratingsentry/update-entry", _parameters }
			};

			var _obj = await this.Session.InvokeAsync(_command);

			return (dynamic)_obj;
		}
	}
}